sdk/resources/arm-resources/package.json

```diff
@@ -1,13 +1,20 @@
 {
   "name": "@azure/arm-resources",
+  "sdk-type": "mgmt",
   "author": "Microsoft Corporation",
-  "description": "ResourceManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "4.2.0",
+  "description": "A generated SDK for ResourceManagementClient.",
+  "version": "30.0.0-beta.1",
```

Comment:
I wonder why jumping from v4 to v30?

Reply:
@deyaaeldeen we will release all track2 js sdks from v30. When we see a package in npm, we can easily know whether the package is track2 js sdk based on version number.